utils/splunk/splunk_helper.py

```diff
@@ -117,6 +117,14 @@ def _do_post(self, path, payload, request_timeout_seconds):
         try:
             resp.raise_for_status()
         except HTTPError as error:
+            if not splunk_ignore_config:
+                self.log.error("Received error response with status {} and body {}".format(resp.status_code, resp.content))
+                raise error
```

Comment:
can we create a test in integrations-core for this? i.e. an exception needs to be thrown when `splunk_ignore_config=False`